.env.sample

```diff
@@ -5,11 +5,25 @@ DATABASE_PASSWORD=metagame_secret
 DATABASE_NAME=metagame-db
 
 HASURA_GRAPHQL_ADMIN_SECRET=metagame_secret
-NEXT_TELEMETRY_DISABLED=1
-
 HASURA_PORT=8080
-NEXT_PUBLIC_INFURA_ID=781d8466252d47508e177b8637b1c2fd
-NEXT_PUBLIC_GRAPHQL_URL=https://api.metagame.wtf/v1/graphql
 
+# packages/web
+NEXT_TELEMETRY_DISABLED=1
+NEXT_PUBLIC_INFURA_ID=781d8466252d47508e177b8637b1c2fd
+NEXT_PUBLIC_GRAPHQL_URL=http://localhost:8080/v1/graphql # use https://api.metagame.wtf/v1/graphql for prod backend
 NEXT_BRIGHTID_APP_URL=https://app.brightid.org
 NEXT_BRIGHTID_NODE_URL=http:%2f%2fnode.brightid.org
+NEXT_DISCORD_BOT_CLIENT_ID=808843072980647986
+NEXT_PUBLIC_VERCEL_URL=http://localhost:3000
+
+# packages/discord-bot
+GRAPHQL_URL=http://localhost:8080/v1/graphql
+FRONTEND_URL=http://localhost:3000
+DISCORD_BOT_TOKEN=
+DISCORD_BOT_CLIENT_ID=
+DISCORD_BOT_CLIENT_SECRET=
+
+# packages/backend
+IMGIX_TOKEN=
+DISCORD_BOT_TOKEN=  # create one at https://discordapp.com/developers/applications/
```

Comment:
Is this required for the backend to run? If so, let's fix that. If not, let's just remove this. Contributors shouldn't need to create their own discord app or get the actual token in order to run locally.